symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/model/ActivityInfo.java

```diff
@@ -28,6 +30,19 @@
   private String description;
 
   /** Whether the bot mention is required or not */
-  private boolean requiresBotMention;
+  private Object uniqueObject;
```

Comment:
We probably want to avoid generic object so we can rely on static typing to prevent errors

Would defining equals/hashcode for activities be a way to achieve what we want?
each activity would have access to its field (whereas activityinfo is just the info and does not hold anything specific to a particular activity)